homeassistant/components/overkiz/sensor.py

```diff
@@ -354,10 +354,16 @@ class OverkizSensorDescription(SensorEntityDescription):
     OverkizSensorDescription(
         key=OverkizState.IO_HEAT_PUMP_OPERATING_TIME,
         name="Heat Pump Operating Time",
+        icon="mdi:home-clock",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        native_unit_of_measurement=TIME_SECONDS,
```

Comment:
Maybe use device class duration?

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes

Reply:
Implemented in follow-up PR.